plugin/src/test/java/com/epam/grandhackathon/deployment/sphere/plugin/BuildVersionMetaDataPublisherTest.java

```diff
@@ -0,0 +1,93 @@
+package com.epam.grandhackathon.deployment.sphere.plugin;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.isA;
+
+import java.io.PrintStream;
+
+import hudson.model.Action;
+import hudson.model.BuildListener;
+import hudson.model.FreeStyleBuild;
+import hudson.model.AbstractBuild;
+import hudson.tasks.BuildStepMonitor;
+
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import com.epam.grandhackathon.deployment.sphere.plugin.action.DynamicVariablesStorageAction;
+
+public class BuildVersionMetaDataPublisherTest {
+
+	@Test
+	public void shouldReturnMonitorServiceBuild() throws Exception {
+		BuildVersionMetaDataPublisher publisher = new BuildVersionMetaDataPublisher();
+		assertEquals("Illegal required monitor service ", publisher.getRequiredMonitorService(), BuildStepMonitor.BUILD);
+	}
+
+	@Test
+	public void shouldOnPerformRejectNullValues() throws Exception {
```

Comment:
Better, but it's not obvious what you're testing, test method should tell me what it's doing and error of failed test should tell me what was wrong - it's like a basic idea of convention to structure tests